File: java/utility/MakeContext.java
package voxgig.bluefinpayconexsdk.utility;

import java.util.Map;

import voxgig.bluefinpayconexsdk.core.Context;

final class MakeContext {

  private MakeContext() {}

  static Context makeContext(Map<String, Object> ctxmap, Context basectx) {
    return new Context(ctxmap, basectx);
  }
}

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BluefinPayconex SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BluefinPayconexMakeContext
{
    public static function call(array $ctxmap, ?BluefinPayconexContext $basectx): BluefinPayconexContext
    {
        return new BluefinPayconexContext($ctxmap, $basectx);
    }
}
